src/actions/classifier.js

```diff
@@ -72,6 +72,13 @@ export function saveAnnotation(task, value) {
   }
 }
 
+export function removeAnnotationValue(task, value) {
+  return (dispatch, getState) => {
+    const workflowID = getState().classifier.currentWorkflowID
+    dispatch(removeState(`classifier.annotations.${workflowID}.${task}`, value))
```

Comment:
Perfect, that is what should happen. Initially, the shortcut task was set to cancel out the answer to the original question, but it seems it is set up correctly here.